Code_Exercises/Exercise_11_In_Order_Queue/solution.cpp

```diff
@@ -28,7 +24,7 @@ class kernel_b_2;
 class kernel_c_2;
 class kernel_d_2;
 
-class usm_selector : public sycl::device_selector {
+class usm_selector {
```

Comment:
Actually this should be just a function now. No need for this complexity, even if it works too of course.